<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20pt;</string>
    </property>
    <property name="text">
     <string>Fichier eleve</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NÂ°inscription</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nom">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pre">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="M">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Masculin</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="F">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Feminin</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>600</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajout</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dessin_cb">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dessin</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="theater_cb">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Theater</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="music_cb">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sports_cb">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sports</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
